replicaset.go

```diff
@@ -52,6 +47,22 @@ var (
 	isReady          = IsReady
 )
 
+// attemptInitiate will attempt to initiate a mongodb replicaset with each of
+// the given configs, returning as soon as one config is successful.
+func attemptInitiate(monotonicSession *mgo.Session, cfg []Config) error {
+	var err error
+	for _, c := range cfg {
+		logger.Infof("Initiating replicaset with config %#v", c)
+		err = monotonicSession.Run(bson.D{{"replSetInitiate", c}}, nil)
+		if err != nil {
+			logger.Infof("Unsuccessful attempt to initiate replicaset: %v", err)
+		} else {
```

Comment:
I've very recently added the else for clarity around the early return on success.